unattended_installer/install_functions/installCommon.sh

```diff
@@ -248,8 +271,8 @@ function installCommon_installPrerequisites() {
             common_logger "--- Dependencies ----"
             for dep in "${not_installed[@]}"; do
                 common_logger "Installing $dep."
-                eval "DEBIAN_FRONTEND=noninteractive apt install ${dep} -y ${debug}"
-                if [  "$?" != 0  ]; then
+                installCommon_aptInstall ${dep}
+                    if [  "$?" != 0  ]; then
```

Comment:
Consider avoiding `$?` use and be more specific, due to `$?` variable being the last command execution result, it must not be the apt install command.

Reply:
In this commit (https://github.com/wazuh/wazuh-packages/pull/1401/commits/78fec1b274e978379457051fd04228a37d38b05f), removed the validation where we gave the apt lock error message since the apt status code 100 is not typical of this error, but any error in apt throws 100 as the status code, so the message is not accurate.

For example, if the version of the package that we pass is incorrect, the status code will also be 100
```
root@ubuntu20:~/wazuh-packages/unattended_installer# apt install -y unzip=124975
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Version '124975' for 'unzip' was not found
root@ubuntu20:~/wazuh-packages/unattended_installer# echo $?
100
```

So the validation we do does not convince me because if we have an error for which we should not wait, we are generating an unnecessary wait for the user